bin/offline-dbg

```diff
@@ -133,6 +142,129 @@ EOF
     echo "**************************************************************"
 }
 
+check_for_OpenFlow_14() {
+    # If no protocol listed, default to atomic bundle txn (enabled by default on ovs 2.8 and on)
+    local hex_ofp_version=$(cat ${CMD_DIR}/ovs-ofctl_-t_5_--version | grep "OpenFlow versions" | grep -oP [0-9]*x[0-9]* | tail -1)
+    local version=${open_flow_protocols[$hex_ofp_version]}
+    if [ -z "$version" ]; then
+        echo " --bundle"
+    else
+        [ ${version#OpenFlow} -ge 14 ] && bundle=" --bundle" || bundle=""
+        echo $bundle
+    fi                    
+}
+
+do_collect-sos() {
+    if [ $# -lt 1 ]; then
+        usage
+        exit 1
+    fi
+    local sos=$1
+
+    local dir_name=`tar -tf $sos | head -1 | cut -f1 -d"/"`
+
+    local sos_files="${dir_name}/sos_commands/rpm ${dir_name}/sos_commands/openvswitch"
+
+    local db_locations="var/lib/openvswitch etc/openvswitch usr/local/etc/openvswitch"
+
+    if test -f "$sos"; then
+        mkdir ${WORKDIR}/sos_report
+        echo "Extracting sos report from '$sos'..."
+    else
+        echo "SOS archive file not found. Please specify a <sosreport>.tar.xz archive"
+    fi
+
+    for dir in $sos_files; do
+        tar -xvf $sos -C ${WORKDIR}/sos_report $dir --strip-components 1 &>/dev/null || error "Could not extract critical directory $dir from $sos"
+    done
+    CMD_DIR=${WORKDIR}/sos_report/sos_commands/openvswitch/
+
+    local db_dir=''
+    for dir in $db_locations; do
+        tar -xvf $sos -C ${WORKDIR}/sos_report ${dir_name}/${dir}/conf.db --strip-components 1 &>/dev/null || true
+        if test -f "${WORKDIR}/sos_report/${dir}/conf.db"; then
+            db_dir=$dir
+            break
+        fi
+    done
+
+    if [ -z "${db_dir-}" ]; then
+        echo "warning: conf.db not found in default locations."
+        echo "If the target cluster has defined \$ovs_dbdir it could be located there"
+        echo "Please add manually using collect-db or add the .db directory to \$db_locations and re-run collect-sos"
```

Comment:
These are things for developers (db_locations is the name of the variable inside the openvswitch script). We can skip that part or just say (or use a more recent sos package)